.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+### Prerequisites
+
+* [ ] Put an X between the brackets on this line if you have checked that your issue isn't already filed: https://github.com/minetest-mods/mob-engine/issues?utf8=%E2%9C%93&q=is%3Aissue
```

Comment:
@BrunoMine If you want go to https://gitter.im/minetest-mob-engine/Admins, I explain to you in Portuguese